src/client/java/teammates/client/scripts/sql/DataMigrationForFeedbackSessionSql.java

```diff
@@ -0,0 +1,74 @@
+package teammates.client.scripts.sql;
+
+import java.time.Duration;
+
+// CHECKSTYLE.OFF:ImportOrder
+import com.googlecode.objectify.cmd.Query;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.entity.FeedbackSession;
+import teammates.storage.sqlentity.Course;
+
+// CHECKSTYLE.ON:ImportOrder
+/**
+ * Data migration class for feddback sessions.
+ */
+@SuppressWarnings("PMD")
+public class DataMigrationForFeedbackSessionSql
+        extends DataMigrationEntitiesBaseScriptSql<FeedbackSession, teammates.storage.sqlentity.FeedbackSession> {
+
+    public static void main(String[] args) {
+        new DataMigrationForFeedbackSessionSql().doOperationRemotely();
+    }
+
+    @Override
+    protected Query<FeedbackSession> getFilterQuery() {
+        return ofy().load().type(teammates.storage.entity.FeedbackSession.class);
+    }
+
+    @Override
+    protected boolean isPreview() {
+        return false;
+    }
+
+    @Override
+    protected void setMigrationCriteria() {
+        // No migration criteria currently needed.
+    }
+
+    @Override
+    protected boolean isMigrationNeeded(FeedbackSession entity) {
+        return true;
+    }
+
+    @Override
+    protected void migrateEntity(FeedbackSession oldEntity) throws Exception {
+        HibernateUtil.beginTransaction();
+        Course course = HibernateUtil.get(teammates.storage.sqlentity.Course.class, oldEntity.getCourseId());
+        HibernateUtil.commitTransaction();
```

Comment:
This transaction is probably causing the migration to be slow, we probably can optimise this by joining courses with its feedback sessions

Reply:
Changed it to use [EntityManager.getReference](https://docs.oracle.com/javaee/7/api/javax/persistence/EntityManager.html#getReference-java.lang.Class-java.lang.Object-).